addons/mail/static/src/components/emoji_grid_search_no_content_view/emoji_grid_search_no_content_view.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.EmojiGridSearchNoContentView" owl="1">
+        <div class="o_EmojiGridSearchNoContentView d-flex flex-column w-100 h-100 justify-content-center align-items-center">
```

Comment:
shouldn't all the flexbox stuff be grouped together?

Reply:
done